trino/client.py

```diff
@@ -826,13 +836,14 @@ def _double_map_func(self):
             else float(val)
 
     def _timestamp_map_func(self, column, col_type):
+        datetime_default_size = 20
```

Comment:
Why 20. I see:

```python
>>> len(datetime.datetime.strftime(datetime.datetime.utcnow(), "%Y-%m-%d %H:%M:%S"))
19
```

I'm probably missing something or this deserves a comment.

Reply:
Done